docs/source/install/updating_installation.rst

```diff
@@ -1,21 +1,22 @@
 .. _updating_installation:
+.. _updating_aiida:
 
-*********************
-Updating installation
-*********************
+**************
+Updating AiiDA
+**************
 
-Before you update your AiiDA installation, first make sure that you do the following:
+Before you update your AiiDA installation, make sure to:
 
 * Stop your daemon by executing ``verdi daemon stop``
 * Create a backup of your database(s) by following the guidelines in the :ref:`backup section<backup>`
 * Create a backup of the ``~/.aiida`` folder (where configuration files are stored)
 
 If you have installed AiiDA manually from a local clone of the ``aiida_core`` repository, skip to the instructions for :ref:`developers <update_developers>`.
-Otherwise, if you have installed AiiDA through ``pip``, you can also update your installation through ``pip``.
-If you installed ``aiida_core`` in a virtual environment make sure to load it first.
-Now you are ready to update your AiiDA installation through ``pip``::
 
-  pip install --upgrade aiida_core
+If you have installed AiiDA through ``pip``, 
+enter the python environment where you've installed AiiDA and run::
+
+  pip install --upgrade aiida-core
```

Comment:
Could you please add here the new migration request message that was added in a recent PR. Tell them that it will be triggered by running any `verdi` command that requires the database and will print the command that needs to be run, i.e. `verdi -p <profile_name> database migrate`. This is now the same for both ORM backends

Reply:
Also I do not think that running `verdi setup` after updating the code is necessary anymore. This was to update daemon config files for celery and supervisor, but since we do not use that anymore, I think we can remove it. This is repeated in the upgrade for developers section and can also be removed